builder/FindGlobals.cmake

```diff
@@ -185,3 +183,52 @@ endfunction()
 function(report_add_target var target)
   set(${ARGV0} ${${ARGV0}} ${ARGV1} CACHE INTERNAL "" FORCE)
 endfunction()
+
+# Defined OS name and version and build info and build commit
+if(APPLE)
+  set(MFX_SYSTEM "MAC")
+elseif(UNIX)
+  execute_process(
```

Comment:
What will happen if lsb_release is not available? I am specifically worry about such flavors as FreeBSD. Can you, please, account for that and at least make sure that we will print out "unknown"?

Reply:
If lsb_release is not available than output will be first string output command.
Search performed by parsing string with target words "Distributor ID:<Tab>" if these words are missing than the output will be first string output command. I will add check 
```
if(NOT "${OUTPUT}" STREQUAL "${MFX_LINUX_NAME}")
  set(MFX_SYSTEM "${MFX_LINUX_NAME}"
```